release-notes/EPM/I-DOCSIS/EPM_7.0.8_Integrated_DOCSIS.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: EPM_7.0.8_Integrated_DOCSIS
+---
+
+# EPM 7.0.8 Integrated DOCSIS - Preview
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be added, modified, or moved to a later release. Check back soon for updates!
+
+## Changes
+
+### Enhancements
+
+#### CISCO CBR-8 CCAP Platform: CCAP TV data added [ID 41139]
+
+Two new tables are now available in the CISCO CBR-8 element: the *Video QAM Channel* table and the *MPEG Output Streams* table. These tables are shown on the *Video Data* page of the element. You can also retrieve the data from these tables in a dashboard or low-code app. If you do not need this TV data for your implementation, leave the *Poll Video Data Tables* parameter set to its default setting *Disabled*, so that this data will not be polled.
+
+#### New OFDMA + LowSplitSCQAM column added in US FN Peak dashboard [ID 41311]
+
+In the US FN Peak dashboard, a new column, *OFDMA + LowSplitSCQAM*, has been added. The values in this column are calculated by adding the highest peak at the same timestamp for the Low Split SCQAM and OFDMA Utilization for a user-defined time period. Note that the Peak OFDMA Utilization used in this calculation is different than the Peak OFDMA Utilization displayed on the dashboard. It is the highest peak in a 24-hour period, without taking the 3 highest peaks like the parameter shown in the dashboard.
+
```

Comment:
Note that the Peak OFDMA Utilization used in this calculation is different than the Peak OFDMA Utilization displayed on the dashboard. Should it be *different from*?

Reply:
Both are possible, but "different from" is indeed more common. (https://dictionary.cambridge.org/grammar/british-grammar/different-from-different-to-or-different-than)